lib/fog/vsphere/models/compute/cdroms.rb

```diff
@@ -0,0 +1,43 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Cdroms < Fog::Collection
+        autoload :Cdrom, File.expand_path('../cdrom', __FILE__)
```

Comment:
It might work but thats not the expected. `autoload` will define a constant on that scope.
So i would expect it to define a constant in the wrong place.